Unix/tools/auth_tests_setup.sh

```diff
@@ -37,34 +40,60 @@ do
 done
 
 #
-# if required env variables don't exist, then quit
+# Setup the NTLM auth file for client and server. They must be different files because 
+# if the differeing permissions, as the server file must be owned by root and the client by
```

Comment:
For now, the file must be owned by the effective user.  If we go to  a non-root user, then it should be owned by that user. Who else would we want it owned by?